apps/web/pages/event-types/index.tsx

```diff
@@ -60,6 +60,8 @@ import SkeletonLoader from "@components/eventtype/SkeletonLoader";
 
 import { ssrInit } from "@server/lib/ssr";
 
+import { UpgradeTeamsBadge } from "../../../../packages/ui/components/badge/UpgradeTeamsBadge";
```

Comment:
```suggestion
import { UpgradeTeamsBadge } from "@calcom/ui/components/badge/UpgradeTeamsBadge";
```

Reply:
Likely not to be honest @emrysal

Reply:
true. its a smart component, better in features

Reply:
but we actually dont wanna use this component at all haha